runtime.md

```diff
@@ -34,22 +34,24 @@ This is provided so that consumers can find the container's configuration and ro
 ```
 
 ## Lifecycle
-
-### Create
-
-Creates the container: file system, namespaces, cgroups, capabilities.
-
-### Start (process)
-
-Runs a process in a container.
-Can be invoked several times.
-
-### Stop (process)
-
-Not sure we need that from runc cli.
-Process is killed from the outside.
-
-This event needs to be captured by runc to run onstop event handlers.
+The lifecycle describes the timeline of events that happen from when a container is created to when it ceases to exist.
+
+1. OCI compliant runtime is invoked by passing the bundle path as argument.
+2. The container's runtime environment is created according to the configuration in config.json.
+   Any updates to config.json after container is running do not affect the container.
+3. The container's state.json file is written to the filesystem under /run/opencontainer/<runtime>/containers/<id>/.
+4. The prestart hooks are invoked by the runtime.
+   If any prestart hook fails, then the container is stopped and the lifecycle continues at step 8.
+5. The user specified process is executed in the container.
+6. The poststart hooks are invoked by the runtime.
+   If any poststart hook fails, then the container is stopped and the lifecycle continues at step 8.
+7. Additional actions such as pausing the container, resuming the container or signaling the container may be performed using the runtime interface.
```

Comment:
I [still][1] think we want to drop pause/resume/signal until we have more clarity on how they will work.  For more background here, especially around complications with shared state on Linux, see wking/oci-command-line-api#3.  Alternatively, if we already have a clear idea of how these will work [in the absence of /run/opencontainer][2], I'd appreciate some comments in the CLI-spec PR explaining what the CLI should look like.

[1]: https://github.com/opencontainers/specs/pull/231#discussion_r45532412
[2]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/q6TYqVZOcX8